src/formatDate.js

```diff
@@ -1,14 +1,41 @@
 'use strict';
 
-/**
+/*
  * @param {string} date
  * @param {string[]} fromFormat
  * @param {string[]} toFormat
  *
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  // split date by 4 position('-') to new array;
+  const dateArr = date.split(fromFormat[3]);
```

Comment:
The delimiter is assumed to be at the fourth position in the `fromFormat` array, but this assumption may not hold true for all input formats. Consider dynamically determining the delimiter based on the actual format string.